shaders/src/shading_parameters.fs

```diff
@@ -9,18 +9,26 @@
 void computeShadingParams() {
 #if defined(HAS_ATTRIBUTE_TANGENTS)
     highp vec3 n = vertex_worldNormal;
+#if defined(MATERIAL_HAS_ANISOTROPY) || defined(MATERIAL_HAS_NORMAL) || defined(MATERIAL_HAS_CLEAR_COAT_NORMAL)
+    highp vec3 t = vertex_worldTangent;
+    highp vec3 b = vertex_worldBitangent;
+#endif
 
 #if defined(MATERIAL_HAS_DOUBLE_SIDED_CAPABILITY)
     if (isDoubleSided()) {
         n = gl_FrontFacing ? n : -n;
+#if defined(MATERIAL_HAS_ANISOTROPY) || defined(MATERIAL_HAS_NORMAL) || defined(MATERIAL_HAS_CLEAR_COAT_NORMAL)
+        t = gl_FrontFacing ? t : -t;
+        b = gl_FrontFacing ? b : -b;
```

Comment:
I'm not sure this is correct because negating all axes of a base will change its handedness. Is it correct to have a left-handed TBN for back faces? Or should only N and one of T or B be negated?

Reply:
fwiw negating just the normal as it is now also flips its handedness.

Reply:
Yes, I realized that too. So is there a test/sample this actually fixes?

Reply:
Yeah - the backfaces on the gltf sample model [NormalTangentTest](https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/NormalTangentTest) when displayed with the gltfio ubershader.

Reply:
Tim, I think we're conflating two bugs, there is a bug with backface culling (#1872), then there's a tangents-generation bug that I think you're attempting to fix.  Maybe you could file a ticket for the tangents-related bug?  Including a screenshot would be helpful too.  We'll probably need to close this PR since it produces a non-orthonormal tangent frame.

Reply:
Oh interesting.  I didn't realize that you were seeing culling issues in #1872.  The issue I'm seeing is that backface normals are flipped without this:
![flipped_normals](https://user-images.githubusercontent.com/16946090/68784521-88e22280-05f1-11ea-857b-168c964eb1f8.png)


I'm not sure how this could create a non-orthonormal tangent frame.  It should still be orthonormal, just rotated differently than it is currently.

Reply:
Note that I genuinely don't know if ending up with a left handed TBN is correct or not. I was just wondering. I say, if it fixes a gltf test without breaking other ones, this seems better than what we currently have.